include/solvers/simulation_parameters.h

```diff
@@ -155,6 +155,15 @@ class SimulationParameters
         throw std::logic_error(
           "Inconsistency in .prm!\n with VOF = true\n use: number of fluids = 2");
       }
+
+    if (multiphysics.vof_parameters.sharpening.type ==
+          Parameters::SharpeningType::adaptative &&
+        not(multiphysics.vof_parameters.conservation.monitoring))
+      {
+        throw std::logic_error(
+          "Inconsistency in .prm!\n in subsection VOF, with sharpening type = adaptative\n "
+          "use: monitoring = true");
```

Comment:
As there are subparameters for the monitoring that should also be chosen (like `id fluid monitored`), I prefer to define it as an error so that the user pay attention that an entire subsection of the prm file might be missing

Reply:
Ok I understand, seems legit.